src/gui/static/src/app/components/layout/hardware-wallet/hw-passphrase-activation-dialog/hw-passphrase-activation-dialog.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, Inject } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163608)
```suggestion
import { Component, Inject } from "@angular/core";
```